supabase/migrations/20260723120547_fix_app_create_storage_rls.sql

```
@@ -0,0 +1,181 @@
+  AND (
+    (
+      (storage.foldername(objects.name))[1] = 'org'
+      AND (storage.foldername(objects.name))[3] IS NOT NULL
+      AND (storage.foldername(objects.name))[3] <> 'logo'
+      AND (
+        (
+          EXISTS (
+            SELECT 1
+            FROM public.apps
+            WHERE apps.owner_org = ((storage.foldername(objects.name))[2])::uuid
+              AND apps.app_id = ((storage.foldername(objects.name))[3])::character varying
```

SQLFluff (4.2.2)

[error] Line is too long (90 > 80).

(LT05)